tools/mercury/README.md

```diff
@@ -0,0 +1,508 @@
+# Mercury
+##interactive Golang Testing System for Qpid Dispatch Router
+
+
+
+<br/>
+
+
+###Requirements
+
+1. the go language on your machine
+2. installed proton and dispatch router software
+
+
+
+<br/>
+
+###Mercury Audience
+
+Mercury is a tool for developers. The user has one or more installed versions of the Dispatch Router + Proton code and wantsto easily set up a complex network including interior routers, edge routers, and clients with nontrivial addressing patterns. Currently the creation of nontrivial networks is difficult enough that it discourages extensive testing during development.
+
+Especially Mercury provides a more interactive form of development testing, in which a developer easily creates a network, and easily iterates through a cycle of altering the network, running it, and seeing results from the run. At the end of such a 'session' all of the developer's actions have been captured and will be reproducible later if desired. The captured session is a runnable script, and can be edited and used as a standardized test.
+
+
+<br/>
+
+
+
+###The C Client
+
+The client is written in C to the proactor interface and has been heavily adapted from an original by Alan Conway. It needs to be built before Mercury will be able to do anything useful. Look at the directory mercury/clients, look at the file in there called "m", adapt it for your system, and run it so you get an executable.
+
+Having Mercury's own client allows it to do things like:
+
+1. throttle send-speed with the "--throttle" argument
+2. Tell the client to form multiple links with multiple "--address" arguments.
+3. Tell it where to send its log files and so on.
+
+
+<br/>
+
+
+###Starting Mercury
```

Comment:
I read the "Starting Mercury" section and was eager to start mercury. I opened the mercury/example_run_script, changed the   MERCURY_ROOT, then launched it. I see - 

☿ error: rn.check_path: Path |dispatch_root| does not exist at |/home/mick/latest/install/dispatch|
Should this "Starting Mercury" section address how this path needs to set *before* running example_run_script ?